Cleaning_Code/cleaning_traffic_upto2018.py

```diff
@@ -30,4 +30,10 @@ def speed_condition(c):
 
 traffic_df["speed_category"] = traffic_df.apply(speed_condition, axis = 1)
 
-traffic_df["comments"] = ""
\ No newline at end of file
+traffic_df["comments"] = ""
```

Comment:
I like the way that this is separated but I think you could put everything in one file if you were so inclined. Not a blocker though.